roles/server/defaults/main.yml

```diff
@@ -97,7 +105,7 @@ metadata: !!null
 
 networkconfig: !!null
 
-pool: 'default'
+pool: "{{omit}}"
```

Comment:
not sure why i changed that. i actually had to create a default pool before. so why dont we check if the "default" pool was already created and create if not, since the pool creation logic is already implemented